lib/cpp/test/TSSLSocketInterruptTest.cpp

```diff
@@ -57,12 +54,6 @@ struct GlobalFixtureSSL
         BOOST_TEST_MESSAGE(boost::format("argv[%1%] = \"%2%\"") % i % master_test_suite().argv[i]);
       }
 
-#ifdef __linux__
-      // OpenSSL calls send() without MSG_NOSIGPIPE so writing to a socket that has
-		// disconnected can cause a SIGPIPE signal...
-		signal(SIGPIPE, SIG_IGN);
```

Comment:
Agree, I would leave that in otherwise applications built on thrift with SSL that do not do this will exit (without a core if I remember correctly, which was really hard to figure out).